index.js

```diff
@@ -5,19 +5,26 @@ const eyeglass = require("eyeglass");
 
 const app = express()
 const port = 1337
+const debug = true
+const proxy = debug
 
 app.use(
   sassMiddleware(
     eyeglass({
       src: __dirname, //where the sass files are
       dest: __dirname, //where css should go
-      debug: true, // obvious
+      debug: debug, // obvious
```

Comment:
Why was this on before? Is this debug flag a requirement for the local proxy?

Reply:
No idea. I retained it since it was already there.